### ISSUE TITLE
Get definitions of grade contexts from library

### ISSUE THREAD

**Issue by musoke** [OpenBeta/open-tacos#797]
**What would you like to be able to do? Please describe.**

Replace the code at https://github.com/OpenBeta/open-tacos/blob/e6d9e1a9d8e70cd6e87a00ff631cd56c11e93899/src/js/grades/Grade.ts#L5-L28 with an import of that in https://github.com/OpenBeta/openbeta-graphql/blob/d44abd624345c52fae6de1c702333131fe1f5cf0/src/GradeUtils.ts#L35-L84 (which itself has a comment saying it should move to sandbag).

This will ease maintenance going forward.

Should probably wait until the relevant code is in sandbag?

**How important is this to you** (Please pick one)

- Nice-to-have

**Comment by actuallyyun** [comment]
I'd like to work on this!

**Comment by actuallyyun** [comment]
On the same note, maybe we should also change the implementation of [GradeBands in opentacos ](https://github.com/OpenBeta/open-tacos/blob/develop/src/js/grades/bandUtil.ts) to using [GradeBands from Sandbag](https://github.com/OpenBeta/sandbag/blob/b6a606bbd532066842bdc8e9aece3f1bdea41d83/src/GradeBands.ts#L5)
This is related to the issue [Grade bands are defined multiply and inconsistently #91](https://github.com/OpenBeta/sandbag/issues/91)

**Comment by musoke** [comment]
I think the plan is to just remove grade bands from sandbag?

**Comment by actuallyyun** [comment]
> I think the plan is to just remove grade bands from sandbag?

Then I must have misunderstood the plan. I thought we want to extract it in sandbag

**Comment by vnugent** [comment]
@musoke should we move grade context to sandbag?

**Comment by musoke** [comment]
> @musoke should we move grade context to sandbag?

I think so, but for simplicity was going to leave it until the LCO push settles down.

Edit: probably not necessary to wait on the sandbag end. @actuallyyun, I will move grade contexts from openbeta-graphql to sandbag so that you can do this one.
